### PR TITLE
redirect to datatable after add a content

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Controller/ContentController.php [hunk 1]
```diff
@@ -107,11 +107,7 @@ public function newAction(Request $request, $contentType)
         if ($this->handleForm($form, $message, $content)) {
             $this->dispatchEvent(ContentEvents::CONTENT_CREATION, new ContentEvent($content));
 
-            return $this->redirect(
-                $this->generateUrl('open_orchestra_backoffice_content_form', array(
-                    'contentId' => $content->getContentId()
-                ))
-            );
+            return $this->render('BraincraftedBootstrapBundle::flash.html.twig');
```
Comment (itkg-ngilain) [inline BackofficeBundle/Controller/ContentController.php]: I'm not sure to understand: when a content is created we now see a page with only a flashbag ?

File: BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div id="flashbag">
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig]: You should give a class instead of an id

**Comment by itkg-rboessel** [comment]
:+1: with class and if test pass after rebase

**Comment by itkg-rboessel** [comment]
:+1:

File: BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="flashbag">
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig]: Isn't there some classes in smartadmin about flashes?

**Pull request merged**